packages/ui-progress/src/ProgressBar/index.tsx

```diff
@@ -126,10 +126,6 @@ class ProgressBar extends Component<ProgressBarProps> {
             css={styles?.htmlProgress}
             max={valueMax}
             value={valueNow}
-            role="progressbar"
-            aria-valuetext={valueText}
-            aria-valuenow={valueNow}
-            aria-valuemax={valueMax}
```

Comment:
These are not needed. I guess they were there because screenreaders years ago did not support `<progress>` natively? Now it works fine without these, e.g. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/progressbar_role#prefer_html